docs/building.md

```diff
@@ -72,36 +72,73 @@ sudo port install "${dependencies[@]}"
 ```
 
 #### Windows
-First you need to install [MSYS2](https://www.msys2.org), then startup "MSYS2 UCRT64" and execute the following
-commands.
+
+ℹ️ Cross-compilation is not supported on Windows. You must build on the target architecture.
```

Comment:
This looks copied from build-deps... we don't use this style in the doxygen docs. See https://docs.lizardbyte.dev/projects/sunshine/master/md_third-party_2doxyconfig_2docs_2source__code.html